sros2.repos

```diff
@@ -0,0 +1,161 @@
+repositories:
```

Comment:
@mikaelarguedas should this be synced with the upstream? This is the current diff:

```diff
--- repositories: (Selection)
+++ (clipboard)
@@ -51,10 +51,6 @@
     type: git
     url: https://github.com/ros2/ament_cmake_ros.git
     version: master
-  ros2/cli_tools:
-    type: git
-    url: https://github.com/ros2/cli_tools.git
-    version: master
   ros2/common_interfaces:
     type: git
     url: https://github.com/ros2/common_interfaces.git
@@ -90,11 +86,15 @@
   ros2/rcl:
     type: git
     url: https://github.com/ros2/rcl.git
-    version: sros2
+    version: master
   ros2/rcl_interfaces:
     type: git
     url: https://github.com/ros2/rcl_interfaces.git
     version: master
+#  ros2/rclc:
+#    type: git
+#    url: https://github.com/ros2/rclc.git
+#    version: master
   ros2/rclcpp:
     type: git
     url: https://github.com/ros2/rclcpp.git
@@ -107,18 +107,46 @@
     type: git
     url: https://github.com/ros2/rcutils.git
     version: master
+  ros2/realtime_support:
+    type: git
+    url: https://github.com/ros2/realtime_support.git
+    version: master
   ros2/rmw:
     type: git
     url: https://github.com/ros2/rmw.git
-    version: sros2
+    version: master
+  ros2/rmw_connext:
+    type: git
+    url: https://github.com/ros2/rmw_connext.git
+    version: master
   ros2/rmw_fastrtps:
     type: git
     url: https://github.com/ros2/rmw_fastrtps.git
-    version: sros2
+    version: master
   ros2/rmw_implementation:
     type: git
     url: https://github.com/ros2/rmw_implementation.git
-    version: sros2
+    version: master
+  ros2/rmw_opensplice:
+    type: git
+    url: https://github.com/ros2/rmw_opensplice.git
+    version: master
+  ros2/robot_model:
+    type: git
+    url: https://github.com/ros2/robot_model.git
+    version: ros2
+  ros2/robot_state_publisher:
+    type: git
+    url: https://github.com/ros2/robot_state_publisher.git
+    version: ros2
+  ros2/ros1_bridge:
+    type: git
+    url: https://github.com/ros2/ros1_bridge.git
+    version: master
+  ros2/ros2cli:
+    type: git
+    url: https://github.com/ros2/ros2cli.git
+    version: master
   ros2/rosidl:
     type: git
     url: https://github.com/ros2/rosidl.git
@@ -131,18 +159,26 @@
     type: git
     url: https://github.com/ros2/rosidl_typesupport.git
     version: master
+  ros2/sros2:
+    type: git
+    url: https://github.com/ros2/sros2.git
+    version: master
   ros2/system_tests:
     type: git
     url: https://github.com/ros2/system_tests.git
-    version: master
-  ros2/sros2:
-    type: git
-    url: https://github.com/ros2/sros2.git
     version: master
   ros2/tinyxml_vendor:
     type: git
     url: https://github.com/ros2/tinyxml_vendor.git
     version: master
+  ros2/tlsf:
+    type: git
+    url: https://github.com/ros2/tlsf.git
+    version: master
+#  ros2/tutorials:
+#    type: git
+#    url: https://github.com/ros2/tutorials.git
+#    version: master
   ros2/urdfdom:
     type: git
     url: https://github.com/ros2/urdfdom.git
```

I don't know enough about this to say if it is so.